plugins/modules/zabbix_script.py

```diff
@@ -0,0 +1,416 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, BGmot
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+module: zabbix_script
+short_description: Gather information about Zabbix proxy
+version_added: 1.6.0
```

Comment:
This has to be next feature release
```suggestion
version_added: 1.7.0
```